RickR2H/joomla-cms
# Removed obsolete CSS for debug position

Pull Request for Issue # .

### Summary of Changes
In PR https://github.com/joomla/joomla-cms/pull/34849 the debug position is removed from the template grid area. So this code is obsolete and can be removed.

### Testing Instructions
Apply the patch and activate debug position. Everything should work. Also on mobile.

### Actual result BEFORE applying this Pull Request

### Expected result AFTER applying this Pull Request

### Documentation Changes Required
